packages/swapper/src/types.ts

```diff
@@ -195,6 +199,16 @@ export type GetTradeQuoteInput =
   | GetEvmTradeQuoteInput
   | GetCosmosSdkTradeQuoteInput
 
+export type GetTradeRateInput =
+  | GetEvmTradeRateInput
+  | GetCosmosSdkTradeRateInput
+  | GetUtxoTradeRateInput
+
+export type GetTradeQuoteInputWithWallet =
```

Comment:
Not sure about naming here - i.e we have `TradeQuoteInput` (a quote or rate), `TradeRateInput` (an explicit rate input) and this guy, which is *explicitly* a quote subset